lib/salus/scanners/yarn_audit.rb

```diff
@@ -184,6 +186,26 @@ def normalize_vuln(vuln)
       vuln['ID'] = id.to_i
     end
 
+    def combine_vulns(vulns)
+      uniq_vulns = {} # each key is uniq id
+
+      vulns.each do |vul|
+        id = vul['ID']
+        if uniq_vulns[id]
+          uniq_vulns[id]['Dependency of'].push vul['Dependency of']
+        else
+          uniq_vulns[id] = vul
+          uniq_vulns[id]['Dependency of'] = [uniq_vulns[id]['Dependency of']]
+        end
+      end
+
+      vulns = uniq_vulns.values
+      vulns.each do |vul|
+        vul['Dependency of'] = vul['Dependency of'].sort.join(', ')
```

Comment:
Can we have any cases where we may have the same "dependency of" listed more than once?  I assume we won't.  If we will we may want a .uniq.sort here.